### DISCUSSION TITLE
question about submission

### DISCUSSION THREAD

**Discussion by billbiancheng** [PaddlePaddle/PaddleSpatial#68] [BaiduKDDCup2022] [answered]
I just used the baseline code you provided (the ones under test-1.zip) to test the submission system, however, the following error was returned - The preparation script, i.e.  'NoneType' object has no attribute 'prep_env'

**Comment by ramber1836** [comment, marked as answer]
This is mainly because your path is not set correctly.
Please follow this discussion:
https://github.com/PaddlePaddle/PaddleSpatial/discussions/65
